### PR TITLE
feat: Consolidate all in interfaces (Debian)

### PR DESCRIPTION
**Pull request by jbe-dw**
This PR is a proposal for #132

### CODE REVIEW

**Comment by jbe-dw** [comment]
This would put all in interfaces file with each config separated with a distinct comment

File: tasks/debian.yml [hunk 1]
```diff
@@ -41,6 +41,19 @@
     group: root
     mode: 0644
   tags: configuration
+  when: not interfaces_merge
+
+- name: Debian | Ensure lo single configuration file
```
Comment (markgoddard) [inline tasks/debian.yml:46]: Needs to go after L63

File: templates/bond_slave_Debian.j2 [hunk 2]
```diff
@@ -1,5 +1,9 @@
-# {{ ansible_managed }}
+{% if not interfaces_merge %}# {{ ansible_managed }}
+{% else %}# Bond Slave - {{ item.0.device }}/{{ item.1 }}
+{% endif %}
 
 auto {{ item.1 }}
 iface {{ item.1 }} inet manual
+{% if ansible_distribution_major_version|int < 11 %}
```
Comment (markgoddard) [inline templates/bond_slave_Debian.j2:7]: `ansible_facts.distribution_major_version`

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: templates/bond_slave_Debian.j2 (hunk 2 above)
Comment (jbe-dw) [reply templates/bond_slave_Debian.j2]: done

**Comment by jbe-dw** [comment]
Thanks for the review

**Review by markgoddard — APPROVED** [review]

**Pull request merged**